board/Microchip_M2S010/BSP/Inc/config.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2006-2020, RT-Thread Development Team
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Change Logs:
+ * Date           Author       Notes
+ * 2020-08-06     whik         first version
+ */
+#ifndef __CONFIG_H__
+#define __CONFIG_H__
+
+#include "mss_gpio.h"
+#include "mss_uart.h"
+
+#include <rthw.h>
+#include <rtthread.h>
```

Comment:
Thanks for your pull request，please remove the code form RT-Thread，Then we can merge this.